File: Conditional.java
// Conditional.java

import java.util.Scanner;

public class Conditional
{
   // main method begins execution 
   public static void main(String[] args)
   {
      // create Scanner to obtain input from command line
      Scanner input = new Scanner(System.in);

      int number1; // first number to compare
      int number2; // second number to compare

      System.out.print("Enter first integer: "); // prompt 
      number1 = input.nextInt(); // read first number  

      System.out.print("Enter second integer: "); // prompt 
      number2 = input.nextInt(); // read second number 
      
      if (number1 == number2) 
         System.out.printf("%d == %d%n", number1, number2);

      if (number1 != number2)
         System.out.printf("%d != %d%n", number1, number2);

      if (number1 < number2)
         System.out.printf("%d < %d%n", number1, number2);

      if (number1 > number2)
         System.out.printf("%d > %d%n", number1, number2);

      if (number1 <= number2)
         System.out.printf("%d <= %d%n", number1, number2);

      if (number1 >= number2)
         System.out.printf("%d >= %d%n", number1, number2);
   } // end main
} // end class 

File: Conditional.cpp
// Conditional.cpp
#include <iostream>
using namespace std;

int main()
{

      int number1; // first number to compare
      int number2; // second number to compare

      cout<<"Enter first integer: "<<endl; // prompt 
      cin>>number1; // read first number  

      cout<<"Enter second integer: "<<endl; // prompt 
      cin>>number2; // read second number 
      
      if (number1 == number2) 
         cout<<number1<< " = "<< number2<<endl;

      if (number1 != number2)
         cout<<number1<< " != "<< number2<<endl;

      if (number1 < number2)
         cout<<number1<< " < "<< number2<<endl;

      if (number1 > number2)
         cout<<number1<< " > "<< number2<<endl;

      if (number1 <= number2)
         cout<<number1<< " <= "<< number2<<endl;

      if (number1 >= number2)
         cout<<number1<< " >= "<< number2<<endl;;
   } // end main
} // end class 
